package.json

```diff
@@ -41,9 +41,15 @@
     "LeanKit"
   ],
   "scripts": {
-    "build": "gulp",
-    "test": "gulp test",
-    "coverage": "gulp coverage"
+    "prebuild": "npm run lint",
+    "build": "webpack --config webpack.config.js",
+    "pretest": "npm run lint",
+    "test": "karma start --no-auto-watch --single-run || true",
```

Comment:
I may have gone a little overboard on that, and I think I found a better way to handle the issue I was trying to fix.  Did you introduce a linting error before trying `npm test`? I based this on a little googlearching (I'm making this a word, starting now) when I found this: https://github.com/npm/npm/issues/6124

However, this one is actually a much better approach: http://stackoverflow.com/questions/26814034/silencing-errors-on-failures-for-npm-run-script.